components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -56,16 +56,18 @@
           <div class ="channel-line">
             <chef-badge *ngIf="service.channel !== ''">{{ service.channel | uppercase }}</chef-badge>
             <chef-badge no-data
-              id="general-channel-tooltip"
+              [id]="'channel-tooltip-' + i"
               [tooltip]="tooltipMessageFor('channel')"
               *ngIf="service.channel === ''">NO CHANNEL</chef-badge>
           </div>
           <div class ="health-line">
             <chef-icon class="{{ service.health_check }}">{{ service.health_check | serviceStatusIcon }}</chef-icon>
-            {{ healthCheckStatus(service) }}
+            {{ healthCheckStatus(service.health_check) }}
           </div>
-          <div class ="timewizard-line">
-            Since {{ service.current_health_since }} ago
+          <chef-tooltip [attr.for]="'health-timestamp' + i">{{ formatTimestamp(service.health_updated_at) }}</chef-tooltip>
```

Comment:
2.) change this to `<chef-tooltip style='--tooltip-color: #222435;' [attr.for]="'health-timestamp' + i">{{ formatTimestamp(service.health_updated_at) }}</chef-tooltip>`

Reply:
I am changing this now but, out of curiosity, why do we prefer the inline style here instead of the css?

Reply:
@Shadae With this change now the unit tests are failing:
```
src/app/page-components/services-sidebar/services-sidebar.component.html
--
  | L67 \|          <chef-tooltip style='--tooltip-color: #222435;' [attr.for]="'health-timestamp-' + i">{{ for...
  | ^ Inline style [  style='--tooltip-color: #222435;' ] cannot be use. (inline-style-disabled)
```